pkg/k8sutil/create.go

```diff
@@ -161,3 +166,67 @@ func parseJSONManifest(data []byte) ([]manifest, error) {
 	}
 	return manifests, nil
 }
+
+// UpdateNamespace updates the namespace.
+func UpdateNamespace(namespace Namespace, kubeconfig []byte) error {
```

Comment:
Also, we should be adding unit tests for new functions.

Reply:
This is covered in e2e tests.